saw-core/src/SAWCore/Term/Certified.hs

```diff
@@ -139,18 +146,72 @@ scTypeOf sc (Term ctx _tm tp) =
      let ctx' = pruneContext (freeVars tp_tp) ctx
      pure (Term ctx' tp tp_tp)
 
--- | Reduce a 'Cterm' to WHNF (see also 'scTypeCheckWHNF').
+-- | Reduce a 'Term' to WHNF using all reductions and conversions
+-- allowed by the SAWCore type system (see also 'scTypeCheckWHNF').
 scWHNF :: SharedContext -> Term -> IO Term
 scWHNF sc (Term ctx tm tp) =
   do tm' <- scTypeCheckWHNF sc tm
      pure (Term ctx tm' tp)
 
+-- | Reduce a 'Term' to WHNF using beta reductions, tuple and record
+-- reductions, recursor reductions, and unfolding definitions.
+scWhnf :: SharedContext -> Term -> IO Term
+scWhnf sc (Term ctx tm tp) =
+  do tm' <- Raw.scWhnf sc tm
+     pure (Term ctx tm' tp)
```

Comment:
Having two functions named `scWHNF` and `scWhnf` is rather confusing, especially since they do different things. Is it feasible to give one (or both) a more descriptive name to make it easier to tell them apart?